src/partials/messenger.navigation.html

```diff
@@ -120,7 +120,7 @@
 <div id="navigation-contacts" class="tab-content" ng-if="ctrl.activeTab == 'contacts'" in-view-container>
     <p class="empty" ng-if="ctrl.contacts().length === 0" translate>messenger.NO_CONTACTS_FOUND</p>
     <ul ng-class="{'hide-inactive': ctrl.hideInactiveContacts()}">
-        <li ng-repeat="contact in ctrl.contacts() | isNotMe | filter:ctrl.searchContact"
+        <li ng-repeat="contact in ctrl.contacts() | isNotMe | isNotHidden | filter:ctrl.searchContact"
```

Comment:
IIRC all of these will run their own filter loop? Wouldn't it make sense to create a combined filter where conditionals can be added in form of a callback?

Something like: `filter(isNotMe, isNotHidden, ...)`

Reply:
Resolved in the other conversation. @lgrahl was right :slightly_smiling_face: